rafthttp/transport.go

```diff
@@ -78,13 +81,25 @@ type Transporter interface {
 	Stop()
 }
 
-type transport struct {
-	roundTripper http.RoundTripper
-	id           types.ID
-	clusterID    types.ID
-	raft         Raft
-	serverStats  *stats.ServerStats
-	leaderStats  *stats.LeaderStats
+// Transport implements Transporter interface. It provides the functionality
+// to send raft messages to peers, and receive raft messages from peers.
+// User should let the handler returned by Handler method in Transport
```

Comment:
User should call Handler method to get a handler to serve requests received from peerURLs. (remove the unnecessary details.)